runconfig/config.go

```diff
@@ -30,6 +31,7 @@ type Config struct {
 	Cmd             *stringutils.StrSlice // Command to run when starting the container
 	Image           string                // Name of the image as it was passed by the operator (eg. could be symbolic)
 	Volumes         map[string]struct{}   // List of volumes (mounts) used for the container
+	CLIVolumeSpecs  map[string]struct{}   // CLI-passed list of volumes (mounts) used for the container
```

Comment:
PS I don't particularly like `RawVolumes`, or at least the `volumes` part of it, as it's more than volumes - it's volumes AND bind mounts. And `Raw` implies (at least to me) something quite different to what it is. It's just the user-supplied -v specs. I do like `spec` though as it's commonly referred to as that in the existing code. JMHO. As I said though, I really don't care, it's just a name....